Islandora/controlled_access_terms
# Fix Taxonomy Term updates not propagating to Fedora

**GitHub Issue**: https://github.com/Islandora/documentation/issues/1417

# What does this Pull Request do?

Adds missing changed mapping to all the rdf mappings:
```
  changed:
    properties:
      - 'schema:dateModified'
    datatype_callback:
      callable: 'Drupal\rdf\CommonDataConverter::dateIso8601Value'
```

# What's new?

* Added changed mapping to rdf mappings
* Does this change require documentation to be updated? No.
* Does this change add any new dependencies? No.
* Does this change require any other modifications to be made to the repository
 (i.e. Regeneration activity, etc.)?  No.
* Could this change impact execution of existing code? No.

# How should this be tested?

- Create a Person (or any other Taxonomy term provided by controlled_access_terms): _The person appears in Fedora_
- Update the person: _The changes **do not appear** in Fedora (and Milliner complains)_
- Apply the PR && import the feature changes
- Update the person again: _The changes **do appear** in Fedora (and no Milliner complaints)_

# Interested parties
@bseeger and @dannylamb
